### PR TITLE
up'ing the max_cache_size on the data producers.

### PR DESCRIPTION
**Pull request by gknisely**

### CODE REVIEW

**Comment by dnesbitt61** [comment]
:shipit:

File: attributes/default.rb [hunk 1]
```diff
@@ -52,7 +52,7 @@
 
 # configuration
 default[:valhalla][:config]                                      = "#{node[:valhalla][:conf_dir]}/valhalla.json"
-if !node[:opsworks] || (node[:opsworks][:layers][:'data-producer'] && node[:opsworks][:instance][:layers].include?('data-producer'))
```
Comment (kevinkreiser) [inline attributes/default.rb:55]: i dont understand this change, what were we hoping to accomplish here? if you wanted to increase the amount of ram the data producer threads use why wouldnt you have changed the equation below

**Comment by gknisely** [comment]
@dnesbitt61 wanted to use the calculation to bump up the max_cache_size.  No agree @kevinkreiser ?

**Comment by kevinkreiser** [comment]
sorry i dont understand the change though. it looks here as if you just neutered the if statement. that will just make it so that the calculation is never used. basically why even have an if statement at all. in fact why not just hardcode the value at this point.

i would have assumed if you wanted to try higher use of ram you'd have to change the calculation part, particularly that `.9` or that `2` in there

**Comment by dnesbitt61** [comment]
on the data producer it appeared the calculation was bypassed - setting the default (1GB). The change I think makes it so the data producers use the calculation as well

**Comment by kevinkreiser** [comment]
it used to work. i wonder if something with the structure of the attributes has changed? @heffergm would know for sure. at any rate, wont this change basically make it also never hits the calculation?

**Comment by dnesbitt61** [comment]
if not an opsworks node it uses the default, otherwise it computes a value is how I read it (but I'm not a chef expert!)

**Comment by kevinkreiser** [comment]
oh i see im looking at it all wrong, the calculation is for the servers. it was indeed intended to set the data producer to one gig.

**Comment by kevinkreiser** [comment]
i guess i completely forgot why i did that in the first place hahaha

**Comment by gknisely** [comment]
@dnesbitt61 @kevinkreiser so all good i assume

**Comment by kevinkreiser** [comment]
Yeah ignore me
Am 25.01.2016 6:07 PM schrieb "Greg Knisely" notifications@github.com:

**Pull request merged**